README.rst

```diff
@@ -5,7 +5,7 @@ coala-bears
 
```

Comment:
If we were to use Fixes", then let's wait till I do away with all the `coala-analyzer` occurences in the repo.  This just fixes for the README.rst file.